packaging/linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: keybase
+version: master
+summary: Crypto for everyone!
+description: |
+  Keybase service and command line client.
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict
+
+apps:
+  keybase:
+    command: keybase
+    plugs: [network, network-bind]
+
+parts:
+  client-go:
+    source: ../../go
+    plugin: go
+    go-importpath: github.com/keybase/client/go
+    go-packages: [github.com/keybase/client/go/keybase]
+    go-buildtags: [production]
+    after: [go]
+  go:
+    source-tag: go1.7.5
```

Comment:
Will merge as is, but I'm curious if there's a way to outboard the Go version here. We have something similar in our Dockerfile in the folder above (which is on 1.7.1 currently). Would be nice to have them read the same version number from somewhere.